### DISCUSSION TITLE
Increasing Camera Resolution

### DISCUSSION THREAD

**Discussion by ShomaPirogov** [Genymobile/feedback#78] [Genymotion Desktop] [answered]
I have some suggestions for improving your services. One of the key features for me is the camera resolution in virtual Android devices. Currently, Genymotion limits the resolution to 480p, but in modern applications and games, as well as other camera-dependent applications, higher resolution may be required.

Therefore, I would like to request the possibility of increasing the camera resolution, at least up to 720p or, even better, to 1080p in your emulator. This capability will allow for more detailed testing and development of applications, especially those heavily reliant on the camera.

For example, I cannot even work with some web applications' scanners that require a higher camera resolution. I have attached a screenshot below ↓
<img width="375" alt="image" src="https://github.com/Genymobile/feedback/assets/148997360/8c4b7398-0fc6-490a-96cc-5f2faa63dafe">

I understand that changing this parameter may affect the emulator's performance and require additional efforts on your part. However, this enhancement will help your users achieve better results with Genymotion and enhance the attractiveness of your product.

Thank you for considering this matter. I would appreciate your consideration and response.

Best regards,
Yuri.

**Comment by agavignet** [comment, marked as answer]
Hi,

We are not currently supporting better resolutions than 480p for cameras.
Some users like you also brought to our attention this issue. We would very much like to solve this, but our tech team already has a heavy roadmap to deliver. In any case, thank you for the reporting this, it helps us prioritize our roadmap accordingly.

Best regards
Genymotion

**Reply by ShomaPirogov** [reply]
Hello,

Thank you for the response and attention to this issue. While I understand the current limitations and workload on your technical team, I appreciate the acknowledgment of the problem. I look forward to future updates or improvements aimed at increasing the camera resolution. Your transparency is valued, and I am confident that your team will be able to address this issue soon and continue to provide quality solutions.

Best regards,
Shoma Pirogov

**Comment by pericotiko** [comment]
The same thing happens to me that this guy needs a camera with better resolution for many applications this resolution falls very short. I hope they can fix it soon

**Comment by jmaneyrol69** [comment]
@ShomaPirogov @pericotiko We have just released Genymotion Desktop 3.8! This new release includes a new camera API which now supports 720p resolution for Android 12 and above. For more information, please refer to the release notes: https://www.genymotion.com/blog/release-note/genymotion-desktop-3-8-0/
